gradle.properties

```diff
@@ -2,4 +2,5 @@
 android.useAndroidX=true
 android.enableJetifier=true
 org.gradle.jvmargs=-Xmx1g
+org.gradle.parallel=true
```

Comment:
Realized people might not be benefiting from this. This is paying more and more dividends as we commit more to multimodule as each one can have tasks run on them in parallel.